update.sh

```diff
@@ -171,13 +171,38 @@ if [ $(getent group dialout) ]; then
 fi
 
 # ensure some dependencies are installed
-ensure_installed() {
-  if ! dpkg --verify "$1" 2>/dev/null; then
-    # Future: batch the installs, if there are multiple
-    apt-get install $1
-  fi
+ensure_dependencies_installed() {
+    : '
+    Description:
+        Ensures that the given list of dependencies are installed on the system.
+
+    Parameters:
+        $@ - An array of package names to be checked and installed if missing.
+
+    Outputs:
+        Installation messages and potential errors if a package is missing.
+
+    Example Use:
+        ensure_dependencies_installed "libfontconfig1" "another_package" "yet_another_package"
+    '
+
+    local dependencies=("$@")  # Convert arguments into an array
+    local to_install=()  # Array to hold packages that need to be installed
+
+    for package in "${dependencies[@]}"; do
+        if ! dpkg -l | grep -q "^ii  $package "; then
```

Comment:
This isn't working reliably (on my x64 machine), it would appear that the presence of multiple architectures of the library is causing confusion
![image](https://github.com/bitfocus/companion-pi/assets/1327476/0e7a5019-993d-4f04-8b78-7977db811f0c)